python/lsst/ts/eas/hvac_model.py

```diff
@@ -245,3 +253,52 @@ async def wait_for_noon(self, *, hvac_remote: salobj.Remote) -> None:
                                 minFanSetpoint=math.nan,
                                 antiFreezeTemperature=math.nan,
                             )
+
+    async def apply_setpoint_at_night(self, *, hvac_remote: salobj.Remote) -> None:
+        """Controls the HVAC setpoint during the night.
+
+        At night time (defined by `DirunalTimer.is_night`) the HVAC
+        AHU setpoint should be applied based on the outside temperature,
+        if the dome is closed. If the dome is open, the HVAC AHUs
+        should not be enabled, and the setpoint should not matter.
+
+        Parameters
+        ----------
+        hvac_remote : salobj.Remote
```

Comment:
Please enclose parameter types in backticks as per the TSSW documentation style guide.